Dockerfile

```diff
@@ -1,24 +1,43 @@
 FROM node:lts-alpine
-ENV NPM_CONFIG_LOGLEVEL warn
 
-# Install "git"
-RUN apk update \
- && apk add alpine-sdk
+ARG VERSION_INFO=noVersion
+
+ENV NPM_CONFIG_LOGLEVEL warn
+ENV PORT=1337
 
 # Set working dir as /app
 WORKDIR /app
 
-# Add sources to /app
-COPY . .
-RUN adduser -S dr4ftuser
-RUN chown dr4ftuser -R .
-USER dr4ftuser
+COPY --chown=node package.json .
+COPY --chown=node package-lock.json .
 
 # Install the dependencies
-RUN npm ci
```

Comment:
Running this at the end, in combination with the `COPY . .` meant that any time any source file changed, the dependencies would need to get reinstalled and the card info would be re-downloaded again regardless of whether it was necessary due to the layer cache invalidation at the `COPY . .` layer. This slowed down repeated iterations of docker builds significantly.